ArduPlane/commands_logic.cpp

```diff
@@ -613,6 +613,31 @@ bool Plane::verify_takeoff()
     }
 }
 
+/*
+  When enabled, if the previous waypoint is VTOL Takeoff set the flag to do the full rate climbout.
+  This will give a climbing transition and reach the cruise waypoint immediately as opposed to
+  calculating the glide slope based on proportion to the waypoint.
+ */
+#if HAL_QUADPLANE_ENABLED
+void Plane::maybe_do_full_rate_climbout()
+{
+    AP_Mission::Mission_Command prev_cmd;
+
+    // Check the previous mission item.
+    bool prev_read_success = (plane.mission.read_cmd_from_storage(
+        plane.mission.get_current_nav_index() - 1, prev_cmd));
```

Comment:
This is most definitely not reliable.  You may have gotten here with a jump of some description.

Please evaluate `get_prev_nav_cmd_id` instead.